### DISCUSSION TITLE
posts/monte_carlo_integral_with_multiple_importance_sampling/

### DISCUSSION THREAD

**Discussion by giscus** [JiayinCao/AGraphicsGuyNotes#13] [Giscus-Comments]
# posts/monte_carlo_integral_with_multiple_importance_sampling/

The book physically based rendering doesn’t spend too much effort explaining MIS, however it does mention it. In order to be more familiar with MIS(Multiple Importance Sampling), I spent some time reading Veach’s thesis. The whole thesis is relatively long, however the chapter about MIS is kind of independent to the other ones. Worth taking some notes in case I forget later.
Monte Carlo Integral Monte Carlo tries to solve integral problem by random sampling.

https://agraphicsguynotes.com/posts/monte_carlo_integral_with_multiple_importance_sampling/

**Comment by CampbellOwen** [comment]
Thanks for the great article! I think there’s a typo when showing the two rules for MIS being unbiased - I think the first one should be the sum of all the weights is 1 if f(x) *not* equal to 0

**Reply by JiayinCao** [reply]
em, interesting.
I heard the same issue from a friend of mine a few days ago as well.

Here is how it looks on my machine.
![image](https://user-images.githubusercontent.com/5715405/207946746-c91cb5eb-f017-41a0-92db-442ffd97e960.png)

Are you talking about this one?

**Reply by CampbellOwen** [reply]
Oh no, that's an unfortunate rendering error :( I opened the post on my Windows machine in Firefox and now I'm seeing the correct figure (same as what you replied with). I originally read it on my ipad in Safari and saw this:
![image](https://user-images.githubusercontent.com/7470459/207952465-e821a8c6-6ea7-474c-9ad9-56844762b521.png)

Glad to hear it's not an actual typo. but unfortunate that Safari changes the meaning so drastically

**Reply by JiayinCao** [reply]
Could you take a peek with Safari again?
I think it should have been fixed.

**Reply by CampbellOwen** [reply]
Looks as expected in safari now, thanks for the quick update! 

Thanks again for writing the article too, I finally understand how multiple importance sampling works!

**Comment by Mephisto405** [comment]
Another typo when describing the cosistent estimator: N -> inf, not x -> inf
Thx for your great articles! I'm from `https://agraphicsguynotes.com/posts/understanding_the_math_behind_restir_di/#preliminary-reading` post :)

**Reply by Mephisto405** [reply]
![image](https://github.com/JiayinCao/AGraphicsGuyNotes/assets/25499931/8677980f-684a-436b-82c2-5375c883f22c)

**Reply by JiayinCao** [reply]
Thanks for the feedback.
This is fixed.
